repos/spack_repo/builtin/packages/openmpi/package.py

```diff
@@ -598,6 +598,8 @@ class Openmpi(AutotoolsPackage, CudaPackage):
 '-Wl,-flat_namespace'.""",
     )
 
+    variant("cray-xpmem", default=False, description="use cray-xpmem instead of xpmem configure flag (if fabrics=xpmem enabled)")
```

Comment:
I think the comment can be put into a `when`?
```suggestion
    variant("cray-xpmem", default=False, when="fabrics=xpmem", description="use cray-xpmem instead of xpmem configure flag")
```